packages/renderer/src/lib/kube/KubernetesIcon.svelte

```diff
@@ -0,0 +1,51 @@
+<script lang="ts">
+import type { Component } from 'svelte';
+
+import ConfigMapSecretIcon from '../images/ConfigMapSecretIcon.svelte';
+import CronJobIcon from '../images/CronJobIcon.svelte';
+import DeploymentIcon from '../images/DeploymentIcon.svelte';
+import IngressRouteIcon from '../images/IngressRouteIcon.svelte';
+import JobIcon from '../images/JobIcon.svelte';
+import KubeIcon from '../images/KubeIcon.svelte';
+import NodeIcon from '../images/NodeIcon.svelte';
+import PodIcon from '../images/PodIcon.svelte';
+import PvcIcon from '../images/PVCIcon.svelte';
+import ServiceIcon from '../images/ServiceIcon.svelte';
+
+interface Props {
+  kind: string;
+  size?: string;
+}
+let { kind, size }: Props = $props();
+
+let Icon = $derived.by(() => {
```

Comment:
```suggestion
let Icon: Component = $derived.by(() => {
```

here, we have an implicit inferring, probably better to be explicit since we have a lot of options in the switch

Reply:
👍🏼 Fixed.